### DISCUSSION TITLE
Touch, copy and paste

### DISCUSSION THREAD

**Discussion by dineshbvadhia** [zauberzeug/nicegui#2636] [Q&A] [answered]
### Question

Say, you have an infinite grid of images. Above the grid is a rectangular box. If an image is touched then a small copy is pasted into the box. The box can contain one or more images at a time. Is this possible?

**Comment by rodja** [comment, marked as answer]
Sure. Write an `on_click` handler which used it's event args to get the source of the clicked image. Then you can create a new image with that source in the box above the image grid.
